docs/source/developers/guide/documentation.rst

```diff
@@ -27,4 +27,65 @@
 
 **************************
 Helping with documentation
-**************************
\ No newline at end of file
+**************************
+
+
+A great way to contribute to the project is to improve
+documentation. 
+
+If you are an Arrow user and you found some docs to be
+incomplete or inaccurate, share your hard-earned knowledge
+with the rest of the community.
+
+If you haven't come across an error in the documentation
+it is hard to find one and going through the docs in search
+for corrections is not a good way to go. In this case you can
+search for an issue that is dealing with the documentation
+on JIRA.
```

Comment:
Perhaps add something about the "good-first-issue" and "good-second-issue" labels? Example search:
https://issues.apache.org/jira/issues/?jql=project%20%3D%20ARROW%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20in%20(good-first-issue%2C%20good-second-issue)%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC

Reply:
Oh, actually it's done in https://github.com/apache/arrow/pull/11807 . Nevermind.